### PR TITLE
[WIP] feat: (platform) Action bar redesign the action-bar component to have ne…

### PR DESCRIPTION
**Pull request by manu-kr**
Adopt new fiori 3 style and redesign component as per the new techspec

#### Please provide a link to the associated issue.

https://github.com/SAP/fundamental-ngx/issues/2405

#### Please check whether the PR fulfills the following requirements

- [ ] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [ ] tests for the changes that have been done

Documentation checklist:
- [ ] update `README.md`
- [ ] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [ ] Documentation Examples
- [ ] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-docs.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(PlatformActionBarDocsComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
+
+    it('should create', () => {
```
Comment (salarenko) [inline apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-docs.component.spec.ts:21]: Missing tests

File: libs/platform/src/lib/components/action-button-group/action-button-group.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ActionButtonGroupComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```
Comment (salarenko) [inline libs/platform/src/lib/components/action-button-group/action-button-group.component.spec.ts:22]: Missing tests

File: libs/platform/src/lib/components/action-bar/action-bar.component.ts [hunk 3]
```diff
@@ -6,42 +6,21 @@ import {
...
      * Actionbar title
      */
-    @Input()
-    title: string;
 
-    /**
-     * flag to set edit mode for renaming the title
-     */
     @Input()
-    editing: boolean = false;
```
Comment (droshev) [inline libs/platform/src/lib/components/action-bar/action-bar.component.ts:44]: If we remove inputs then this is a breaking change => the PR should start with `feat:` and document all the breaking changes

File: apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-docs.component.spec.ts (hunk 1 above)
Comment (manu-kr) [reply apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-docs.component.spec.ts]: Removed, as documentation not require unit testing

File: libs/platform/src/lib/components/action-button-group/action-button-group.component.spec.ts (hunk 2 above)
Comment (manu-kr) [reply libs/platform/src/lib/components/action-button-group/action-button-group.component.spec.ts:22]: As of now ActionButtonGroup is not have any functionality implementation, and we are using this for content loading, so doesn't require any unit test case.

**Comment by stefanoScalzo** [comment]
Screen Shot 2020-04-27 at 3 07 18 PM
Documentation has duplicate description

File: libs/platform/src/lib/components/action-bar/action-bar.component.ts [hunk 4]
```diff
@@ -6,42 +6,21 @@ import {
...
+import { BaseComponent } from '../base';
 @Component({
-    host: {
-        '(document:click)': 'outSideClick($event)'
-    },
     selector: 'fdp-action-bar',
     templateUrl: './action-bar.component.html',
     styleUrls: ['./action-bar.component.scss'],
-    changeDetection: ChangeDetectionStrategy.OnPush
+    changeDetection: ChangeDetectionStrategy.OnPush,
```
Comment (stefanoScalzo) [inline libs/platform/src/lib/components/action-bar/action-bar.component.ts:15]: remove comma

File: apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-contextual-menu-example.component.ts [hunk 5]
```diff
@@ -3,71 +3,14 @@ import { Component, OnInit, ViewEncapsulation } from '@angular/core';
...
-                label: 'test',
-                type: 'main',
-                priority: 6,
-                callback: () => {
-                    alert('test');
-                }
-            }
-        ];
-    }
+    ngOnInit() {}
```
Comment (JKMarkowski) [inline apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-contextual-menu-example.component.ts:11]: Could you remove unused code ?

File: apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-with-back-button-example.component.ts [hunk 6]
```diff
@@ -2,33 +2,14 @@ import { Component, OnInit, ViewEncapsulation } from '@angular/core';
...
-            },
-            {
-                label: 'Cancel',
-                type: 'negative',
-                priority: 2,
-                options: 'emphasized'
-            }
-        ];
-    }
+    ngOnInit() {}
```
Comment (JKMarkowski) [inline apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-with-back-button-example.component.ts:10]: Could you remove unused code ?

File: apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-with-description-example.component.ts [hunk 7]
```diff
@@ -1,35 +1,15 @@
...
-            },
-            {
-                label: 'Cancel',
-                type: 'negative',
-                priority: 2,
-                options: 'emphasized'
-            }
-        ];
-    }
+    ngOnInit() {}
```
Comment (JKMarkowski) [inline apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-with-description-example.component.ts:10]: Could you remove unused code ?

File: apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-header/platform-action-bar-header.component.ts [hunk 8]
```diff
@@ -3,9 +3,9 @@ import { Component, OnInit } from '@angular/core';
...
     selector: 'app-action-bar-header',
     templateUrl: './platform-action-bar-header.component.html',
-    styleUrls: ['./platform-action-bar-header.component.scss']
+    styleUrls: ['./platform-action-bar-header.component.scss'],
 })
-export class PlatformActionbarHeaderComponent implements OnInit {
+export class PlatformActionBarHeaderComponent implements OnInit {
     constructor() {}
 
     ngOnInit() {}
```
Comment (JKMarkowski) [inline apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-header/platform-action-bar-header.component.ts:11]: Could you remove unused code ?

File: apps/docs/src/app/platform/platform-documentation.routes.ts [hunk 9]
```diff
@@ -32,57 +32,57 @@ export const ROUTES: Routes = [
...
-                    { path: 'api', component: ApiComponent, data: { content: API_FILES.radioGroup } }
-                ]
-            }
-        ]
-    }
+                    { path: 'api', component: ApiComponent, data: { content: API_FILES.radioGroup } },
+                ],
+            },
+        ],
+    },
```
Comment (JKMarkowski) [inline apps/docs/src/app/platform/platform-documentation.routes.ts:87]: Could you add latest prettier changes, that prevent from putting commas at the end of property?

**Comment by manu-kr** [comment]
Closing this PR due to rebase issue , Creating new one with incorporating review comments.

**Pull request closed without merging**